python/cudf/cudf/dataframe/categorical.py

```diff
@@ -45,15 +45,114 @@ def codes(self):
         else:
             return Series(data)
 
-    def set_categories(self, new_categories):
+    def to_string(self):
+        if self.ordered:
+            return "[{}]".format(" < ".join(self.categories.to_array()))
+        return "[{}]".format(", ".join(self.categories.to_array()))
```

Comment:
This should be an internal method I think? I assume this is used in conjunction with printing the Series to print the categories, but there's no public API for this in Pandas.

Reply:
Yeah, it's called by the Series `to_string()` implementation here: https://github.com/rapidsai/cudf/blob/722dc49d8df5c93aa5a724f115212c6ab0d1b67d/python/cudf/cudf/dataframe/series.py#L506-L510

I assumed `to_string` was a python builtin or convention, since that's what the Series was doing. Should we remove it?

Reply:
Nope, it's just a Pandas API, if we move this to `def _to_string(self)` I would be perfectly fine with it